_includes/contact_form.html

```diff
@@ -425,10 +425,14 @@ <h3>{{ site.data.[page.lang].settings.contact_page.support_request_form.title }}
       site.data.[page.lang].settings.contact_page.support_request_form.help_text }}
     </p>
 
+    <b>
+      {{ site.data.[page.lang].settings.contact_page.support_request_form.required_text }} (<abbr title="{{ site.data.[page.lang].settings.contact_page.required }}" class="usa-hint usa-hint--required">*</abbr>).
+    </b>
+
     <!-- START: Topic -->
-    <label class="usa-label" for="00NU0000004z903"
-      >{{ site.data.[page.lang].settings.contact_page.support_request_form.labels.topic }}</label
-    >
+    <label class="usa-label" for="00NU0000004z903">
+      {{ site.data.[page.lang].settings.contact_page.support_request_form.labels.topic }} (<abbr title="{{ site.data.[page.lang].settings.contact_page.required }}" class="usa-hint usa-hint--required">*</abbr>).
```

Comment:
The partners site doesn't use the parentheses. Should these be aligned?

https://login.gov/partners/contact/